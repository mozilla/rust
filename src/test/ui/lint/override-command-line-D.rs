// compile-flags: -D warnings
// check-pass

fn main() {
    #[warn(unused)]
    let a = 5;
    //~^ WARNING: unused
}
