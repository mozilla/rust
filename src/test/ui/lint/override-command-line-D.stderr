warning: unused variable: `a`
  --> $DIR/override-command-line-D.rs:6:9
   |
LL |     let a = 5;
   |         ^ help: if this is intentional, prefix it with an underscore: `_a`
   |
note: the lint level is defined here
  --> $DIR/override-command-line-D.rs:5:12
   |
LL |     #[warn(unused)]
   |            ^^^^^^
   = note: `#[warn(unused_variables)]` implied by `#[warn(unused)]`

warning: 1 warning emitted

